documentation/Use Case Scenarios/Completing Habit Scenario.txt

```diff
@@ -1,10 +1,16 @@
 Context: The user has loaded the application and has added a scenario
 Primary Flow:
 1.	The user selects a habit
-2.	The user selects the circle next to the habit
-3.	The habit is marked as complete 
-4.	The habit is sent to the bottom of the list with the other completed habits.
-5.	The user receives coins for successfully completing the task.
-6.	The coin amount increases by the coins gained.
+2.	The user selects the check box next to the habits they want to complete
+3.	The user is prompted to ensure they want to complete the habits
+4.	The user selects "yes" confirming they want the habits marked as complete
+5.	The habits are marked as complete 
```

Comment:
Check boxes should not bring up a confirmation box every time. Users should be able to checks multiple habits, then press a button to complete them. Once the button is pressed, it should ask if the user is sure.